corehq/form_processor/reprocess.py

```diff
@@ -48,7 +48,14 @@ def reprocess_unfinished_stub(stub, save=True):
         return
 
     assert form.is_normal
-    result = _reprocess_form(form, save)
+    try:
+        result = _reprocess_form(form, save)
+    except:
+        if save:
+            stub.queued = False
+            stub.save()
```

Comment:
if there's a db error here, or the task gets killed (WorkerLostError), we can end up with ones that are stuck in a queued state but not actually queued, so won't be processed.

as an alternative to this, we could just update the enqueuing timeout: https://github.com/dimagi/commcare-hq/blob/1c8b1b7ff96d62b05bf76af856d165fb030ad330/hqscripts/generic_queue.py#L71